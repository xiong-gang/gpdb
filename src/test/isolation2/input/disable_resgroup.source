-- for all removed resgroups the cgroup dirs should also be removed
! ls -d @cgroup_mnt_point@/cpu/gpdb/*/;
! ls -d @cgroup_mnt_point@/cpuacct/gpdb/*/;

-- reset the GUC and restart cluster.
-- start_ignore
! gpconfig -r gp_resource_manager;
! gpstop -rai;
-- end_ignore

SHOW gp_resource_manager;

-- reset admin_group concurrency setting
ALTER RESOURCE GROUP admin_group SET concurrency 10;
