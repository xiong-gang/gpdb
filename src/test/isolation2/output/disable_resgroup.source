-- for all removed resgroups the cgroup dirs should also be removed
! ls -d @cgroup_mnt_point@/cpu/gpdb/*/;
@cgroup_mnt_point@/cpu/gpdb/6437/
@cgroup_mnt_point@/cpu/gpdb/6438/

! ls -d @cgroup_mnt_point@/cpuacct/gpdb/*/;
@cgroup_mnt_point@/cpuacct/gpdb/6437/
@cgroup_mnt_point@/cpuacct/gpdb/6438/


-- reset the GUC and restart cluster.
-- start_ignore
! gpconfig -r gp_resource_manager;

! gpstop -rai;
-- end_ignore

SHOW gp_resource_manager;
gp_resource_manager
-------------------
queue              
(1 row)

-- reset admin_group concurrency setting
ALTER RESOURCE GROUP admin_group SET concurrency 10;
ALTER
