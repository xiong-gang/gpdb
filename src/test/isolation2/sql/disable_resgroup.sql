-- for all removed resgroups the cgroup dirs should also be removed
! ls -d /sys/fs/cgroup/cpu/gpdb/*/;
! ls -d /sys/fs/cgroup/cpuacct/gpdb/*/;

-- reset the GUC and restart cluster.
-- start_ignore
! gpconfig -r gp_resource_manager;
! gpstop -rai;
-- end_ignore

SHOW gp_resource_manager;

-- reset admin_group concurrency setting
ALTER RESOURCE GROUP admin_group SET concurrency 10;
