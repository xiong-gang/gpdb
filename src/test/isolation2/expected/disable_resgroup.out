-- for all removed resgroups the cgroup dirs should also be removed
! ls -d /sys/fs/cgroup/cpu/gpdb/*/;
/sys/fs/cgroup/cpu/gpdb/6437/
/sys/fs/cgroup/cpu/gpdb/6438/

! ls -d /sys/fs/cgroup/cpuacct/gpdb/*/;
/sys/fs/cgroup/cpuacct/gpdb/6437/
/sys/fs/cgroup/cpuacct/gpdb/6438/


-- reset the GUC and restart cluster.
-- start_ignore
! gpconfig -r gp_resource_manager;

! gpstop -rai;
-- end_ignore

SHOW gp_resource_manager;
gp_resource_manager
-------------------
queue              
(1 row)

-- reset admin_group concurrency setting
ALTER RESOURCE GROUP admin_group SET concurrency 10;
ALTER
